### PR TITLE
Move `sndfile-resample` to sndfile-tools package

### LINKED ISSUE #53: sndfile-resample in both libsamplerate and sndfile-tools
Hi @erikd 
We are getting file collisions with the new 1.04 release:
https://bugs.gentoo.org/645258

Which package is the authoritative source on `sndfile-resample`? I guess this should also be fixed in either one of the repos.

### PR DESCRIPTION
**Pull request by SoapGentoo**
* Fixes https://bugs.gentoo.org/645258
Fixes #53

### CODE REVIEW

**Comment by SoapGentoo** [comment]
@erikd ping

File: src/resample.c [hunk 1]
```diff
@@ -71,17 +62,19 @@ main (int argc, char *argv [])
 	for (k = 1 ; k < argc - 2 ; k++)
 	{	if (strcmp (argv [k], "--max-speed") == 0)
 			max_speed = SF_TRUE ;
+		else if (strcmp (argv [k], "--no-normalize") == 0)
+			normalize = 0 ;
 		else if (strcmp (argv [k], "-to") == 0)
 		{	k ++ ;
-			new_sample_rate = parse_int_or_die (argv [k], "sample rate") ;
+			new_sample_rate = atoi (argv [k]) ;
```
Comment (erikd) [inline src/resample.c]: The `parse_int_or_die` function has *very* different semantics to `atoi`.

File: src/resample.c [hunk 2]
```diff
@@ -71,17 +62,19 @@ main (int argc, char *argv [])
...
 			}
```
Comment (erikd) [inline src/resample.c]: The `parse_int_or_die` function has *very* different semantics to `atoi`.

I thin the semantics is important and I don't mind if you copy over the `parse_int_or_die` function.

**Comment by SoapGentoo** [comment]
@erikd `parse_int_or_die` brought back

**Comment by erikd** [comment]
Thanks!

**Pull request merged**